implicits/src/com/rallyhealth/weepickle/v1/implicits/MacroImplicits.scala

```diff
@@ -31,7 +32,458 @@ object MacroImplicits {
     c.Expr[T](q"${c.prefix}.FromTo.join(${c.prefix}.macroTo, ${c.prefix}.macroFrom)")
   }
 
```

Comment:
what in here is different than before? There's a lot that appears the exact same

Reply:
They _should_ be the same, just moved from `Macros.scala` to `MacroImplicits.scala` under private members.

Reply:
That's right. Apart from that, there is some light refactoring around setting/using the boolean flags for omitting stuff, and I put some of the methods in a more logical order (based on visibility, usage, and abstractness).